notebooks/semantic_search/cohere_wikipedia_ingest.ipynb

```diff
@@ -145,7 +145,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "! adb config create rag --from-json --active "
+    "! adb config create --from-json --active "
```

Comment:
WHat's --active? Do you mean activate? That's the command and this would be different if we left it at active.

Reply:
Plus activate indicates an action to do

Reply:
@bovlb resolution here?

Reply:
`--active` is the option provided by `adb config create` and is different from `adb config activate`.  I'd have no objection to merging them, as I often mistype this myself.  I suggest we add `--activate` as an alias for `--active`.  This isn't really an issue with this PR.